internal/test/wmcb/wmcb_test.go

```diff
@@ -87,11 +86,41 @@ type pkgInfo struct {
 	shaType string
 }
 
+// interface to populate pkInfo structs
+type initPkgInfo interface {
+	// set fields for packageInfo struct
+	initializePkgInfo() error
+	// get checksum file content
+	getSHAFileContent(url string) (string, error)
+	// get SHAvalue for the package
+	getSHAValue(url string) (string, error)
+}
+
 type wmcbFramework struct {
 	// TestFramework holds the instantiation of test suite being executed
 	*e2ef.TestFramework
-	// cniPlugins contains information about the CNI plugin package
-	cniPlugins pkgInfo
+	// contains array of packages to be downloaded
```

Comment:
I think this comment needs to be corrected since you are using map instead of array now